bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -100,6 +100,8 @@ spec:
       - --namespace=$(POD_NAMESPACE)
       - --destination-dir=/etc/kubernetes/static-pod-certs
       - --tls-server-name-override=localhost-recovery
+      - --caches-synced-file=/var/run/informer-caches-synced
+      - -v=2
```

Comment:
why 2 not 4?

Reply:
the default loglevel is historically set to 2, did we change it in 4.x?